backend/src/common/util/consts.go

```diff
@@ -40,15 +40,20 @@ const (
 	// It captures whether the name of the owning ScheduledWorkflow.
 	LabelKeyWorkflowScheduledWorkflowName = constants.FullName + "/scheduledWorkflowName"
 
-
-	LabelKeyWorkflowRunId = "pipeline/runid"
+	LabelKeyWorkflowRunId               = "pipeline/runid"
 	LabelKeyWorkflowPersistedFinalState = "pipeline/persistedFinalState"
 
 	// LabelKeyWorkflowEpoch is a Workflow annotation key.
 	// It captures the the name of the Run.
 	AnnotationKeyRunName = "pipelines.kubeflow.org/run_name"
 
-	AnnotationKeyIstioSidecarInject = "sidecar.istio.io/inject"
-	AnnotationValueIstioSidecarInjectEnabled = "true"
+	AnnotationKeyIstioSidecarInject           = "sidecar.istio.io/inject"
+	AnnotationValueIstioSidecarInjectEnabled  = "true"
 	AnnotationValueIstioSidecarInjectDisabled = "false"
+
+	// AnnotationKeyCacheQualified is a workflow annotation key.
+	// It captures whether this run will go through cache service.
```

Comment:
Could you call out what 'go through cache service means"?

Reply:
Cache service will filter pod with annotation: cache_enabled: "true". Also update the code comment.